Cargo.toml

```
@@ -43,12 +43,12 @@ hyper-openssl = { version="0.9", optional=true }
 openssl = { version="0.10", optional=true }
 
 [dev-dependencies]
-env_logger = "0.8"
+env_logger = "0.9"
 # Required for examples to run
 futures = "0.3.1"
-tokio = { version="1.0", features=["macros", "rt-multi-thread"] }
-clap = { version = "3.0.0-beta.5", features = ["derive"] }
-pretty_assertions = "0.7"
+tokio = { version="1.21", features=["macros", "rt-multi-thread"] }
```

One nit here, lets make this a full version `"1"`.

```suggestion
tokio = { version="1", features=["macros", "rt-multi-thread"] }
```